packages/core-ethereum/src/ethereum.ts

```diff
@@ -148,12 +148,22 @@ export async function createChainWrapper(
     durations.minutes(15)
   )
 
-  let gasPrice: number | BigNumber
-  if (networkInfo.gasPrice) {
-    const [gasPriceValue, gasPriceUnit] = networkInfo.gasPrice.split(' ')
-    gasPrice = ethers.utils.parseUnits(gasPriceValue, gasPriceUnit)
+  // comment out legacy tx 0 in favor of EIP-1559
+  // let gasPrice: number | BigNumber
+  // if (networkInfo.gasPrice) {
+  //   const [gasPriceValue, gasPriceUnit] = networkInfo.gasPrice.split(' ')
+  //   gasPrice = ethers.utils.parseUnits(gasPriceValue, gasPriceUnit)
+  // } else {
+  //   gasPrice = await provider.getGasPrice()
+  // }
+
+  let defaultMaxFeePerGas: BigNumber
+  if (networkInfo.network == 'xdai') {
+    defaultMaxFeePerGas = ethers.utils.parseUnits('10', 'gwei')
```

Comment:
Should this be add to the environments config instead? That way we can define it per chain as well.

Reply:
Agree. We can remove the entry of `gas_price` as it's not longer required in EIP1559 txs

Reply:
removed `gas_price` and `gas_multiplier`